src/components/CardList/CardList.jsx

```diff
@@ -1,5 +1,6 @@
 import "./CardList.css";
 import Card from "./../UI/Card";
+import Spinner from "../UI/Spinner";
```

Comment:
Everything is fine. however, logically it would be more correct to add this functionality to the Card component itself, since you apply new logic specifically to it. There you can even not use Card as nested component but to use an export statement to add it:
```
export default hocSpinner(Card, ... <props if needed> ...);
```
Not critical, you can fix and deliver with the following PR.